taxi/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    country = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f"{self.name}, {self.country}"
```

Comment:
You are overusing f-string. It's not a big issue, but it would be better to simply return the attribute without wrapping it in an f-string. For example: `return self.name`. This is more about code style and readability (Checklist item 3).